validate-differential-expression/README.md

```diff
@@ -0,0 +1,79 @@
+# refine.bio Examples: Validate Differential Expression
+
+These notebooks compare the differential expression results from a refine.bio dataset to an non-refine.bio dataset that both have a similar experimental set up.
+
+You may obtain differential expression results from your own data or a collaborator’s data.
+It is good practice to validate your findings in another dataset that measures a similar tissue with a similar experimental design.
+You can use refine.bio datasets and the steps in this example to quickly check for agreement with your results.
+
+**About the Example Workflow**
+
+The main notebook for this example is [gene_DE_validate](https://alexslemonade.github.io/refinebio-examples/validate-differential-expression/gene_DE_validate.nb.html).
+This notebook compares the differential expression results of the two datasets:  
+
+1) An author-processed dataset's [differential expression results](https://alexslemonade.github.io/refinebio-examples/validate-differential-expression/author_processed_DE.nb.html).  
+2) A refine.bio-processed dataset's differential expression results - which are carried out within [the notebook](https://alexslemonade.github.io/refinebio-examples/validate-differential-expression/gene_DE_validate.nb.html).  
+
+The comparison in this example results in a Venn diagram that illustrates the overlap of the two datasets' differentially-regulated genes.
+For this example workflow, the gene expression datasets and sample metadata are stored in a `data/` directory.
+
+### Validating your own differential expression results with refine.bio data
+
+If you'd like to adapt an example to include data you've obtained from [refine.bio](https://www.refine.bio/), and your own dataset's differential expression results, we recommend following these steps below.
+
+#### Step 1) Find a refine.bio dataset
+
+Find and download a [refine.bio](https://www.refine.bio/) dataset that has a similar set up and metadata to your own differential expression analysis.
+Use the search bar to look for datasets with key terms.
+Filters can help you find datasets with similar platforms.
+
+#### Step 2) Set up your data
+
+Place the refine.bio data and metadata TSV files, along with your own differential expression results you'd like to compare them to in the `data/` directory (if you need further instruction on obtaining differential expression results in the first place, we recommend following our [other example workflow](https://alexslemonade.github.io/refinebio-examples/differential-expression/gene_DE.nb.html)).
+Within the R notebook, you will need to change all the `file.path` references to the file names of your new refine.bio datasets and your own results files.
```

Comment:
Here, you're mentioning the R Notebook here before you have the context of the (software) requirements.